worldguard-bukkit/src/main/java/com/sk89q/worldguard/bukkit/session/BukkitSessionManager.java

```diff
@@ -60,8 +60,7 @@ public void resetAllStates() {
     @EventHandler
     public void onPlayerProcess(ProcessPlayerEvent event) {
         // Pre-load a session
-        LocalPlayer player = WorldGuardPlugin.inst().wrapPlayer(event.getPlayer());
-        get(player).initialize(player);
+        get(WorldGuardPlugin.inst().wrapPlayer(event.getPlayer()));
```

Comment:
why did you change that? it doesn't initialize the player anymore?

Reply:
@Joo200, initialization occurs during session creation, it is created when you try to get it from the cache. So there seems to be no need to specifically call the initialization method from here, instead you can try to create a session if it doesn't exist.

https://github.com/EngineHub/WorldGuard/blob/78498491ddf94f2fce376614e17556e5ff8f6327/worldguard-core/src/main/java/com/sk89q/worldguard/session/AbstractSessionManager.java#L178